docs/UserGuide.md

```diff
@@ -278,4 +296,5 @@ _Details coming soon ..._
 | **Sort**        | Coming Soon                                                                                                                                                                                                                                                                |
 | **Transaction** | `transaction coy/COMPANY_NAME g/GOODS q/QUANTITY pr/PRICE d/BUY` <br/> e.g `transaction coy/McDonalds g/apples q/100 pr/1.5 d/buy` <br/> `transaction coy/COMPANY_NAME g/GOODS q/QUANTITY pr/PRICE d/SELL` <br/> e.g. `transaction coy/KFC g/Chickens q/50 pr/2.55 d/SELL` |
 | **User Guide**  | `user_guide`                                                                                                                                                                                                                                                               |
+>>>>>>> 78b6ac43588d9cbbfef5505a01a268dd1c451564
```

Comment:
Please remove this line after resolving the merge conflicts.

Reply:
Okay will do thank you :)